samples/dnn/fcn_semsegm.cpp

```diff
@@ -50,8 +50,10 @@ static void colorizeSegmentation(const Mat &score, const vector<cv::Vec3b> &colo
     const int cols = score.size[3];
     const int chns = score.size[1];
 
-    cv::Mat maxCl(rows, cols, CV_8UC1);
-    cv::Mat maxVal(rows, cols, CV_32FC1);
+    cv::Mat maxCl=cv::Mat::zeros(rows, cols, CV_8UC1);
+    cv::Mat maxVal=cv::Mat::zeros(rows, cols, CV_32FC1);
```

Comment:
`maxVal` will be completely rewrited by the line 55 so it doesn't make sense to previously set it to zero.
Also Mat(rows, cols, CV_32FC1, cv::Scalar(FLT_MAX)) could be used here to initialize the matrix with given value on creation.

Reply:
Sure, agree. Thanks.